### PR TITLE
S02-208 . Francine-update-new-file

### PR DESCRIPTION
**Pull request by francine-oss**
# WELCOME TO DEVOPS-EASY-LEARNING PULL REQUEST SYSTEM 
# Description of the change
S02-208 . Francine-update-new-file
> Detailed description here

## Checks and balances

- [ ] Tests added and run
- [ ] (External) Interfaces documented
- [ ] Has security implications (describe below)

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing
functionality to not work as expected)

## Related stories, issues and pulls

> Links to Pivotal Tracker stories, JIRA tickets or other pull requests here

## Security considerations

> Describe any security impact this change may have

### CODE REVIEW

**Review by fjen53 — APPROVED** [review]
Everything looks good. Good job

**Review by RKemvou — APPROVED** [review]
ok
miss one line

File: file1 [hunk 1]
```diff
@@ -0,0 +1,2 @@
+I love DevOps, i am an engineer and i am training myself.
+Achille and Jen are a good teacher.
```
Comment (RKemvou) [inline file1:2]: good job francine

**Review by RKemvou — CHANGES_REQUESTED** [review]

File: file1 (hunk 1 above)
Comment (fjen53) [reply file1:2]: well done

**Comment by igor581** [comment]
good job keep doing it

**Pull request closed without merging**